docs/en/query_language/functions/uuid_function.md

```diff
@@ -0,0 +1,29 @@
+# generateUUIDv4 function
+
+Generates [UUID](../../data_types/uuid.md) of version 4.
```

Comment:
"version 4" could have been a link to https://tools.ietf.org/html/rfc4122#section-4.4